--- ristretto-0.8.2/src/Makefile.am~	2016-10-11 13:27:44.000000000 +0000
+++ ristretto-0.8.2/src/Makefile.am	2018-06-10 20:40:50.002019187 +0000
@@ -23,6 +23,7 @@
 	main.c
 
 ristretto_CFLAGS = \
+	$(DBUS_GLIB_CFLAGS) \
 	$(GTK_CFLAGS) \
 	$(GLIB_CFLAGS) \
 	$(GTHREAD_CFLAGS) \
@@ -41,6 +42,7 @@
 	-DLOCALEDIR=\"$(localedir)\"
 
 ristretto_LDADD = \
+	$(DBUS_GLIB_LIBS) \
 	$(GTK_LIBS) \
 	$(GLIB_LIBS) \
 	$(GTHREAD_LIBS) \
