--- ristretto-0.0.21/ristretto.desktop.in~	2008-05-24 15:05:39.000000000 +0200
+++ ristretto-0.0.21/ristretto.desktop.in	2010-05-11 12:39:16.282352288 +0200
@@ -8,6 +8,6 @@
 Terminal=false
 Type=Application
 X-MultipleArgs=false
-Categories=GTK;Graphics;Viewer;
+Categories=X-XFCE;GTK;Graphics;Viewer;
 StartupNotify=true
 MimeType=image/png;image/gif;image/jpeg;image/bmp;image/x-ico;image/x-pixmap;image/tiff;
